app/templates/events/createEvent.html

```diff
@@ -7,7 +7,13 @@
     {% set page_title = 'Create a ' + eventData["program"].programName %}
 
   {% elif template.tag == 'single-program' %}
-    {%set page_title = 'Create Event for ' + eventData["program"].programName %}
+    {% if eventData["program"].programName == 'CELTS-Sponsored Event' %}
+      {%set page_title = 'Create a ' + eventData["program"].programName %}
+    {% elif eventData["program"].programName == 'Adopt-a-Grandparent' %}
```

Comment:
Can we future-proof this to check if the program starts with A, E, I, O, or U, and use a/an appropriately?